sys-tuner/src/main.rs

```diff
@@ -94,6 +94,16 @@ fn tune_kernel_udp_buffers_and_vmmap() {
 
     // increase mmap counts for many append_vecs
     sysctl_write("vm.max_map_count", "500000");
+
+    // Delay writes (and thereby reduce them by being overlapping and discarded)
+    // actually hitting the hardware as much as possible.
+    // This is mainly for AccountsDB and snapshot creation IO.
+    // Moreover, we don't generally care about durability not much.
+    // Reference: https://unix.stackexchange.com/a/41831/364236
+    sysctl_write("vm.dirty_ratio", "90");
+    sysctl_write("vm.dirty_background_ratio", "90");
+    sysctl_write("vm.dirty_expire_centisecs", "36000");
+    sysctl_write("vm.dirty_writeback_centisecs", "36000");
```

Comment:
I might need to fine-tune these values. The value is just picked arbitrarily.

Reply:
I've been somewhat hesitant to take this route for fear of ledger corruption and loss of log messages.  Are these knobs available per block-device/file-system?

Reply:
oh, yeah. loss of log might hurt us...

But this couldn't happen unless `solana-validator` manages to kill the entire linux kernel (before killed by OOM) or power outrage/hardware malfunction starts. I think we don't care the logs in these cases.

And, I don't think these knobs isn't available per block-device/file-system.

Reply:
Also, remember that log is preserved in the dirty page cache even if the validator mysteriously crashes as long as the `write` syscall succeeds. And eventually the linux kernel will flush those dirty pages of logs into block storage.

Reply:
> ledger corruption 

I don't think the tuning of these parameters can increase the risks. Writes are usually buffered (but far less delay) without these tuning already. I think RocksDB properly does submit write barrier io reqs if deemed necessary and the underlying fs/block device manages that by journalling.
Anyway, I'm talking based on in-theory. We're just started to rolling in these tuning just recently...